configuration.md

```diff
@@ -21,5 +22,39 @@ The DSpace server will have a whitelist of properties that can be retrieved by t
 }
 ```          
 
-* 200 OK - if the operation succeed
-* 404 Not found - if the property doesn't exist or isn't configured to be retrieved
\ No newline at end of file
+* 200 OK - if the operation succeeded
+* 404 Not found - if the property doesn't exist or isn't configured to be exposed
+
+## Search methods
+**/api/config/properties/search/isExposed**
```

Comment:
I'm not completely sure about the path `isExposed` but I'm not against it. Just adding this comment to share my thought. It was looking strange to me at a first look but I haven't found any direction in our naming convention https://github.com/DSpace/RestContract?tab=readme-ov-file#on-the-naming-of-endpoints

I found that we have a lot of diversity in the way that search methods have been named so far
- some start with findBy...
- other start with byXXX
- allmost all seems to use the CamelCase convention (that we decide to avoid for the resource collection endpoints)